modules/integration/pom.xml

```diff
@@ -41,11 +41,11 @@
     </licenses>
 
     <modules>
-        <!--module>test-common</module>
```

Comment:
Please check with Product team and enable selected modules. AFAIK, they have disabled certain modules to fix bugs.